docs/Compound-Index.md

```diff
@@ -66,6 +66,19 @@ db.people
   .toArray();
 ```
 
+## Matching Multiple Values
+To find specific people using both their first and last name, we can use [WhereClause.anyOf()](/docs/WhereClause/WhereClause.anyOf()). This allows us to have multiple criteria for on the properties that are part of the compound index. The syntax should be as follows:
```

Comment:
...Criteria for on the properties... The 'on' is not needed.

Reply:
Whoops! I'll fix that and also add a section to the anyOf as you suggested, it would probably be useful for someone trying to get that result but not knowing about compound indices.